accounts-db/src/accounts_index.rs

```diff
@@ -1473,12 +1491,26 @@ impl<T: IndexValue, U: DiskIndexValue + From<T> + Into<T>> AccountsIndex<T, U> {
                         match r_account_maps
                             .insert_new_entry_if_missing_with_lock(pubkey, new_entry)
                         {
-                            InsertNewEntryResults::DidNotExist => {}
-                            InsertNewEntryResults::Existed(other_slot) => {
+                            InsertNewEntryResults::DidNotExist => {
+                                num_did_not_exist += 1;
+                            }
+                            InsertNewEntryResults::Existed {
+                                other_slot,
+                                location,
+                            } => {
                                 if let Some(other_slot) = other_slot {
                                     duplicates_from_in_memory.push((other_slot, pubkey));
                                 }
                                 duplicates_from_in_memory.push((slot, pubkey));
+
+                                match location {
+                                    ExistedLocation::InMem => {
+                                        num_existed_in_mem += 1;
+                                    }
+                                    ExistedLocation::OnDisk => {
+                                        num_existed_on_disk += 1;
+                                    }
+                                }
```

Comment:
this is _adding_ new accounting, not just batching existing?

Reply:
oh n/m i guess these are deferred from the update calls removed in the last diff hunk?